addons/hr/static/tests/tours/hr_employee_flow.js

```diff
@@ -27,3 +27,46 @@ registry.category("web_tour.tours").add('hr_employee_tour', {
         run: 'click',
     },
 ]});
+
+registry.category("web_tour.tours").add("hr_candidate_tour", {
+    url: "/odoo",
+    steps: () => [
+        {
+            content: "Open Recruitment App",
+            trigger: '.o_app[data-menu-xmlid="hr_recruitment.menu_hr_recruitment_root"]',
+            run: "click",
+        },
+        {
+            content: "Open Applications menu",
+            trigger: 'button.dropdown-toggle:contains("Applications")',
+            run: "click",
+        },
+        {
+            content: "Go to Candidates list",
+            trigger: 'a.dropdown-item:contains("Candidates")',
+            run: "click",
+        },
+        {
+            content: "Click on candidate",
+            trigger: '.o_kanban_record:contains("Extended Test Candidate")',
+            run: "click",
+        },
+        {
+            content: "Candidate form should open",
+            trigger: ".o_form_view", // Wait for form view container
+        },
+        {
+            trigger: 'button[title="Toggle Studio"]',
+            run: "click",
+        },
+        {
+            trigger: ".nav-link.o_web_studio_view",
```

Comment:
What if studio is not installed ?
Iff the problem really came from studio and the test relied on hr to be present,
the test should have been in studio with condition on whether hr was installed. But those cases are extremely rare.